src/api/huoa.js

```diff
@@ -0,0 +1,5 @@
+import axios from 'axios';
```

Comment:
I dont love this file's naming convention 

`api.js` would be a better name

Reply:
done!